elf/rtld.c

```diff
@@ -1384,7 +1386,8 @@ of this helper program; chances are you did not intend to run this program.\n\
     GL(dl_rtld_map).l_tls_modid = _dl_next_tls_modid ();
 
   /* If we have auditing DSOs to load, do it now.  */
-  bool need_security_init = true;
+  /* Occlum note: Disable security_init, or ldso will crash at _dl_setup_stack_chk_guard, because _dl_random is NULL */
```

Comment:
Should we initialize the _dl_random with sgx_get_rand()?

Reply:
done